api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java

```diff
@@ -268,8 +270,13 @@ public void deletePatientIdentifierType(PatientIdentifierType patientIdentifierT
 			criteria.add(Restrictions.in("location", locations));
 		}
 		
-		// TODO add junit test for getting by patients
+
+		
 		if (!patients.isEmpty()) {
+
+		
+		if (patients.size() > 0) {
```

Comment:
Why do you think this is better than calling isEmpty()? This can be actually very inefficient as you'll have to compute the size of the list which might be expensive.